.github/templates/linux_binary_build_workflow.yml.j2

```diff
@@ -77,11 +77,7 @@ jobs:
     !{{ binary_env(config) }}
     steps:
       !{{ common.setup_ec2_linux() }}
-      - name: Clone pytorch/pytorch
-        uses: actions/checkout@v2
-        with:
-          path: pytorch
-          submodules: recursive
+      !{{ common.checkout_pytorch("recursive") }}
```

Comment:
This won't work correctly since the checkout needs to be into the `pytorch` directory and not the main workspace

Reply:
Perhaps we should just add an argument to this to change directory for checkout if given a second argument

Reply:
Something like

```
!{{ common.checkout_pytorch("recursive", "pytorch") }}
```

Reply:
Updated, checking how if clean step would work as well